esmeta-ignore.json

```diff
@@ -33,7 +33,6 @@
   "GetPrototypeFromConstructor",
   "GetThisValue",
   "HostEnqueuePromiseJob",
-  "INTRINSICS.DefaultTimeZone",
```

Comment:
I admit I wasn't sure why this was here, but removing it didn't seem to break anything, so instead of renaming to the new name, I just deleted this line. Hope that's OK!

Reply:
There was a bit of imprecision at the ESMeta's part and the new abstract operation has no such issues. 

Nothing to worry about.